config/settings.yml

```diff
@@ -12,7 +12,7 @@ workflow:
   url: 'http://localhost:3001'
 
 staging_location: '/dor/workspace'
-
+globus_location: '/globus/sdr_ingest/qa/uploads/'
```

Comment:
I'd expect this path in our QA shared_configs. Is there a different path we want to default to? Maybe this one is fine? 🤷🏻‍♂️